### ISSUE TITLE
Don't run duplicate builds on Travis

### ISSUE THREAD

**Issue by danielbachhuber** [pantheon-systems/solr-power#99]
One of these isn't necessary:

![image](https://cloud.githubusercontent.com/assets/36432/17002450/2cb11a74-4e80-11e6-904e-4a9a4edea104.png)

**Comment by greg-1-anderson** [comment]
Drush fixes this by limiting the branches built as follows:
```
branches:
  only:
    - master
    - 8.x
    - 7.x
    - 6.x
    - 5.x
    - /^[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+.*$/
```

Note that the 'branches' directive also applies to tags; including semver tags allows tests to run on releases.

This project only has one `dev` branch (master), and uses two-digit version numbers, so perhaps this would work:

```
branches:
  only:
    - master
    - /^[[:digit:]]+\.[[:digit:]]+.*$/
```

**Comment by ataylorme** [comment]
Fixed in [50e4a05cfd10205a88a8f7e674f38c9cd1d115b5](https://github.com/pantheon-systems/solr-power/pull/106/commits/50e4a05cfd10205a88a8f7e674f38c9cd1d115b5)
